### ISSUE TITLE
Boxes slowly eat up disk

### ISSUE THREAD

**Issue by glacials** [glacials/splits-io#546]
We just had the site go down because the production box ran out of disk space! Manifested as a 500 error that said
```
An unhandled lowlevel error occurred. The application logs may have details.
```
on a blank white page. The last deploy to production was 7 days ago and deploys replace boxes entirely, so that's probably how long it takes to have disk space creep up to full. We should find out what's growing (logs? cache? other?) and automatically prune it once in a while so disk space used is stable.

**Comment by BatedUrGonnaDie** [comment]
Rack mini profiler writes to files by default, which may be a cause.  I'll try switching it to redis to see if that helps.

**Comment by BatedUrGonnaDie** [comment]
Closed by #554
